.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -0,0 +1,415 @@
+              --flush-ct \
+              --test '!no-missed-tail-calls' \
+              --sysdump-hubble-flows-count=1000000 --sysdump-hubble-flows-timeout=5m \
+              --sysdump-output-filename "cilium-sysdump-${{ matrix.name }}-<ts>" \
+              --junit-file "cilium-junits/${{ env.job_name }} (${{ join(matrix.*, ', ') }}).xml" \
+              --junit-property github_job_step="Run tests upgrade 1 (${{ join(matrix.*, ', ') }})"
+
+            EXTRA_CONNECTIVITY_TEST=""
+            if [ "${{ matrix.test-flow-interrupts }}" == "true" ]; then
+              # Create pods which establish long lived connections. It will be used by
+              # subsequent connectivity tests with --include-conn-disrupt-test to catch any
+              # interruption in such flows.
+              ./cilium-cli connectivity test --include-conn-disrupt-test --conn-disrupt-test-setup
+              EXTRA_CONNECTIVITY_TEST=" --include-conn-disrupt-test"
+            fi
+
+            # Upgrade
+
+            if [ -z "${{ env.upgrade_cilium_to_vsn }}" ]; then
+              CILIUM_CLI_MODE=helm ./cilium-cli upgrade \
+                ${{ steps.vars.outputs.cilium_main_image_settings }} \
+                ${{ steps.vars.outputs.cilium_install_defaults }}
+            else
+              CILIUM_CLI_MODE=helm ./cilium-cli upgrade \
+                --version ${{ env.upgrade_cilium_to_vsn }} \
+                ${{ steps.vars.outputs.cilium_install_defaults }}
+            fi
+
+            ./cilium-cli status --wait
+            kubectl get pods --all-namespaces -o wide
+            kubectl -n kube-system exec daemonset/cilium -- cilium status
+
+            ./cilium-cli connectivity test --include-unsafe-tests --collect-sysdump-on-failure \
+              \$EXTRA_CONNECTIVITY_TEST \
+              --flush-ct \
+              --test '!no-missed-tail-calls' \
+              --sysdump-hubble-flows-count=1000000 --sysdump-hubble-flows-timeout=5m \
+              --sysdump-output-filename "cilium-sysdump-${{ matrix.name }}-<ts>" \
+              --junit-file "cilium-junits/${{ env.job_name }} (${{ join(matrix.*, ', ') }}).xml" \
+              --junit-property github_job_step="Run tests upgrade 2 (${{ join(matrix.*, ', ') }})"
+
+            # --flush-ct interrupts the flows, so we need to set up again.
+            ./cilium-cli connectivity test --include-conn-disrupt-test --conn-disrupt-test-setup
+
+            # Dowgrade
+
+            CILIUM_CLI_MODE=helm ./cilium-cli upgrade \
+              --version ${{ env.upgrade_cilium_from_vsn }} \
+              ${{ steps.vars.outputs.cilium_install_defaults }} \
+              \$EXTRAS_V13
+
+            ./cilium-cli status --wait
+            kubectl get pods --all-namespaces -o wide
+            kubectl -n kube-system exec daemonset/cilium -- cilium status
+
+            ./cilium-cli connectivity test --test '!check-log-errors' --include-unsafe-tests --collect-sysdump-on-failure \
```

Comment:
I'm guessing we have `--test '!check-log-errors'` here because of a bug. Is there an issue for it?

Reply:
Too much copy paste

Reply:
Came too quick to the conclusion. https://github.com/cilium/cilium-cli/issues/1858 :-(